net/fetcher.go

```diff
@@ -91,11 +91,29 @@ func (bsf *BitswapFetcher) GetBlocks(ctx context.Context, cids []cid.Cid) ([]*ty
 		return nil, err
 	}
 
-	blocks, err := sanitizeBlocks(ctx, unsanitized, bsf.validator)
+	filBlocks, err := sanitizeBlocks(ctx, unsanitized, bsf.validator)
 	if err != nil {
 		return nil, err
 	}
-	return blocks, nil
+
+	// Hackity hack hack for getting messages and receipts
+	// TODO delete this and rebase off of integrated graphsync fetcher
```

Comment:
Before merging, this must either go away or contain a link to a github issue.

Reply:
Thx for the reminder.  This is going away after I rebase on the integrated graphsync fetcher